src/components/Content/Content.js

```diff
@@ -9,19 +9,69 @@ import {
   Modal,
   Typography,
 } from '@material-ui/core'
+import {
+  withStyles,
+} from '@material-ui/core/styles';
 import GitHubIcon from '@material-ui/icons/GitHub'
 import TwitterIcon from '@material-ui/icons/Twitter'
 import LinkedInIcon from '@material-ui/icons/LinkedIn'
+import TextField from '@material-ui/core/TextField'
 import Link from '@material-ui/core/Link'
 import useStyles from './Content.styles'
 import { AwesomeButtonSocial } from 'react-awesome-button'
 import 'react-awesome-button/dist/styles.css'
 import Profile from '../Profile'
 
+const SearchTextField = withStyles((theme) => {
```

Comment:
Extract this `SearchTextField` functionality to some other file in that directory. Create a sperate file for this, if needed.